app/scripts/background.js

```diff
@@ -788,11 +788,11 @@ browser.runtime.onInstalled.addListener(({ reason }) => {
 function setupSentryGetStateGlobal(store) {
   global.getSentryState = function () {
     const fullState = store.getState();
-    const debugState = maskObject(fullState, SENTRY_STATE);
+    const debugState = maskObject({ metamask: fullState }, SENTRY_STATE);
     return {
       browser: window.navigator.userAgent,
       store: debugState,
-      version: global.platform.getVersion(),
+      version: platform.getVersion(),
```

Comment:
`global` is not defined in the background script, but we do import the ExtensionPlatform and instantiate it in this file